markup.go

```diff
@@ -170,7 +170,19 @@ func If(cond bool, markup ...MarkupOrComponentOrHTML) MarkupOrComponentOrHTML {
 	if cond {
 		return List(markup)
 	}
-	return nil
+	var result []MarkupOrComponentOrHTML
+	for _, m := range markup {
+		if m == nil {
+			continue
+		}
+		switch m.(type) {
+		case *HTML, Component:
+			result = append(result, Tag("noscript"))
```

Comment:
I think in this case, it is Vecty internally which handles `nil`, so we would just write:

```
result = append(result, nil)
```

i.e. we shouldn't use `noscript` here directly, but rather let Vecty handle the `nil` -> `noscript` translation itself.